### PR TITLE
DOCS-2599 Compute Apdex and Custom Performance Indicators with RUM Data Guide 

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds a guide about computing your Apdex score with RUM data.

### Motivation

DOCS-2599

### Preview

https://docs-staging.datadoghq.com/alai97/real-user-monitoring-apdex-score-guide/real_user_monitoring/guide/compute-apdex-with-rum-data

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/guide/compute-apdex-with-rum-data.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+## Prerequisites
+
+- Your web or mobile application instrumented with the RUM SDK. To set up instrumentation, see [RUM Browser Monitoring][3], [RUM Android Monitoring][4], and [RUM iOS Monitoring][5].
+- Events from your application available in Datadog.
+
+## Compute an Apdex score
+
+In the example below, you can use the View RUM events and the Largest Contentful Paint performance metric to calculate Apdex. 
+
+With a threshold of `T = 2 sec` and the minimum frustrating latency of `4T = 8 sec`, compute three queries with a formula in a Query Value widget to visualize your Apex score on a quick graph.
```
Comment (alai97) [inline content/en/real_user_monitoring/guide/compute-apdex-with-rum-data.md:40]:
```suggestion
Let's use a hypothetical threshold of `T = 2 sec`. The minimum frustrating latency is `4T = 8 sec`. Compute three queries with a formula in a Query Value widget to visualize your Apex score on a quick graph.
```

**Review by amina196 — APPROVED** [review]

File: content/en/real_user_monitoring/guide/compute-apdex-with-rum-data.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+{{< img src="real_user_monitoring/guide/quick-graph.png" alt="Apdex score in a quick graph" style="width:100%;">}}
+
+### JSON configuration
+
+Your quick graph contains the Apdex performance indicator for your applications in the past day. 
+
+1. On the upper right, select `1d` Past 1 Day to include data collected over the past day.
+2. Click the **JSON** tab next to **Edit**.
+
```
Comment (kayayarai) [inline content/en/real_user_monitoring/guide/compute-apdex-with-rum-data.md:78]:
```suggestion
To access the JSON code for this graph, click the **JSON** tab next to **Edit**.

```

File: content/en/real_user_monitoring/guide/compute-apdex-with-rum-data.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+## Additional visualizations and Apdex scores
+
+In the example above, the Apdex score is relevant to the View RUM events and Largest Contentful Paint performance metric.  
+
+You can also calculate other Apdex scores with the following methods:
+
+- To see the Apdex score trend over time, select `Timeseries` instead of `Query Value` in **Select your visualization**.
+- To compute the Apdex score for a specific application, add an additional `@application.id` query and update your formula.
+- To compute the Apdex score with another RUM performance metric such as First Contentful Paint, replace `@view.LargestContentfulPaint` with `@view.FirstContentfulPaint` in the query.
```
Comment (kayayarai) [inline content/en/real_user_monitoring/guide/compute-apdex-with-rum-data.md:152]:
```suggestion
- To compute the Apdex score with another RUM performance metric such as First Contentful Paint, replace `@view.LargestContentfulPaint` with `@view.FirstContentfulPaint` in the queries.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Looking good! A few edits, mostly simplifying things. Let me know if you aren't sure what I'm getting at in any of them.

File: content/en/real_user_monitoring/guide/compute-apdex-with-rum-data.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+4. Under **Select a visualization**, click **Query Value**. A query value widget appears.
+4. On the top right corner, select **Past 1 Day**. By default, the widget displays in Global Time.
+5. Enter a name for your graph, such as `Apdex Score`.
+6. Click **Export** and choose between a dashboard or notebook. Optionally, click **Copy to Clipboard**.
+7. Under **Export graph**, select the dashboard or notebook you want to export your quick graph to and click **Export**. Optionally, click **New Dashboard** to create a dashboard with this quick graph. 
+
+#### Query A
+
+1. In **Graph your data**, select `RUM` as the data source for query `a` and enter `@view.largest_contentful_paint:<2s`.
+2. Click **Update query** in the dropdown menu. The `Largest Contentful Paint:<2s` query appears next to `RUM` for query `a`.
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/real_user_monitoring/guide/compute-apdex-with-rum-data.md:56]:
```suggestion
2. Press Enter or click **Update query** in the dropdown menu. The `Largest Contentful Paint:<2s` query appears next to `RUM` for query `a`.
```

File: content/en/real_user_monitoring/guide/compute-apdex-with-rum-data.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+#### Query B
+
+1. To create query `b`, click **+ Add Query**.
+2. Select `RUM` as the data source for query `b` and enter `@view.largest_contentful_paint:[2s TO 8s]`.
+3. Click **Update query** in the dropdown menu. The `Largest Contentful Paint:[2s - 3s]` query appears next to `RUM` for query `b`.
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/real_user_monitoring/guide/compute-apdex-with-rum-data.md:62]:
```suggestion
3. Press Enter or click **Update query** in the dropdown menu. The `Largest Contentful Paint:[2s - 3s]` query appears next to `RUM` for query `b`.
```

File: content/en/real_user_monitoring/guide/compute-apdex-with-rum-data.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+
+#### Query C
+
+1. To create query `c`, click **+ Add Query**.
+2. Select `RUM` as the data source for query `c` and enter `@Type:view`.
+3. Click **Update query** in the dropdown menu. The `Type:view` query appears next to `RUM` for query `c`.
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/real_user_monitoring/guide/compute-apdex-with-rum-data.md:68]:
```suggestion
3. Press Enter or click **Update query** in the dropdown menu. The `Type:view` query appears next to `RUM` for query `c`.
```

**Review by kayayarai — APPROVED** [review]
💥

**Pull request merged**